src/arm/radxa_rock-3c.c

```diff
@@ -10,25 +10,25 @@
 #include <stdlib.h>
 #include <string.h>
 #include <sys/mman.h>
-#include "arm/radxa_rock3c.h"
+#include "arm/radxa_rock-3c.h"
 #include "common.h"
 
 #define DT_BASE "/proc/device-tree"
 
-#define PLATFORM_NAME_RADXA_ROCK_3_MODEL_C "Radxa ROCK3 Model C"
-
 #define MAX_SIZE 64
 
-const char* radxa_rock3c_serialdev[MRAA_RADXA_ROCK_3C_UART_COUNT] = { "/dev/ttyS3", "/dev/ttyS4", "/dev/ttyS5", "/dev/ttyS9" };
+const char* radxa_rock3c_serialdev[MRAA_RADXA_ROCK_3C_UART_COUNT] = { "/dev/ttyS2", "/dev/ttyS3", "/dev/ttyS4", "/dev/ttyS5", "/dev/ttyS9" };
```

Comment:
```suggestion
const char* radxa_rock_3c_serialdev[MRAA_RADXA_ROCK_3C_UART_COUNT] = { "/dev/ttyS2", "/dev/ttyS3", "/dev/ttyS4", "/dev/ttyS5", "/dev/ttyS9" };
```
函数名要把rock3c改成rock_3c